platform/pulpcore/app/models/storage.py

```diff
@@ -115,13 +116,35 @@ def mkdir(path):
 
         Args:
             path (str): Absolute directory path.
+
+        Raises:
+            OSError: On error.
         """
         try:
             os.makedirs(path)
```

Comment:
`exist_ok=True` does what we want here. It raises `FileExistsError` on files, but not on directories.

Reply:
Nice.